src/App.js

```diff
@@ -1,11 +1,11 @@
 import { theme } from "./assets/theme";
 import { ThemeProvider } from "@mui/material";
-import ToDoItem from "./templates/to-do-list/to-do-item/ToDoItem";
+import ToDoList from "./templates/to-do-list/ToDoList";
```

Comment:
I would name it in all styles. For example, I do not like the idea of having totally the same names like this:
**to-do-list**/**ToDoList**. It is totally the same.

Reply:
The difference is just camel case.